README_new.md

```diff
@@ -0,0 +1,617 @@
+# Orion Context Broker
+
+This is the code repository for the Orion Context Broker, the reference implementation of the Publish/Subscribe Context Broker GE.
+
+This project is part of FIWARE.
+
+* [FIWARE Site](http://www.fiware.org)
+* [FIWARE Catalogue for Orion](http://catalogue.fiware.org/enablers/publishsubscribe-context-broker-orion-context-broker)
+
+Any feedback on this documentation is highly welcome, including bugs, typos
+or things you think should be included but aren't. You can use [github issues](https://github.com/telefonicaid/fiware-orion/issues/new) to provide feedback.
+
+For documentation previous to Orion 0.23.0 please check the old manuals at FIWARE public wiki:
```

Comment:
Fixed in de740476a6006285a458cff382e091e1c9c191f5 and 217e8111c2e2b1ca24380ba30e4dfd75f3109301